src/recordlinker/schemas/tuning.py

```diff
@@ -90,3 +93,36 @@ def from_tuning_job(cls, job: TuningJob, request: fastapi.Request) -> typing.Sel
         """
         url: str = str(request.url_for("get-tuning-job", job_id=job.id))
         return cls(**job.model_dump(), status_url=pydantic.HttpUrl(url))
+
+
+@dataclasses.dataclass
+class TuningPair:
+    """
+    A pair of PIIRecords that are used for training a model.
+    """
+    record1: PIIRecord
+    record2: PIIRecord
+    sample_used: typing.Optional[int] = None  # the number of records sampled from to produce the pair
+
+    @classmethod
+    def from_data(cls, record1: dict, record2: dict, sample_used: int | None = None) -> typing.Self:
+        """
+        Contruct a TuningPair from raw PII data dictionaries.
+        """
+        return cls(
+            record1=PIIRecord.from_data(record1),
+            record2=PIIRecord.from_data(record2),
+            sample_used=sample_used,
+        )
+
+
+@dataclasses.dataclass
+class TuningProbabilities:
+    """
+    A dict of feature specific probabilities that two records are likely to belong to
+    the same cluster given a collection of TuningPairs. Also included are the number,
+    count, of TuningPairs analyzed and an optional parameter for sample size used.
```

Comment:
This docstring isn't quite accurate (it confused me on first read because I was trying to figure out why we had the probability that two records belong to the same class). When I got to prob_calc, I realized what this data class is for though, so there's an easy fix if we just borrow the language used in those docstrings (helps with both consistency and correctness). I would do something like:

```
A dictionary of class-conditional likelihoods separated by feature. For a given feature F, this holds the probability that a pair of patient records in the same tuning class (i.e. a true-match pair or a non-match pair) will have the same field value in F.
```